app/src/main/java/org/oppia/android/app/customview/ContinueButtonView.kt

```diff
@@ -51,4 +64,16 @@ class ContinueButtonView @JvmOverloads constructor(
       viewModel.animateContinueButton.removeObserver(continueButtonAnimationObserver)
     }
   }
+
+  override fun onAttachedToWindow() {
+    try {
+      super.onAttachedToWindow()
+
+      val viewComponentFactory =
+        FragmentManager.findFragment<Fragment>(this) as ViewComponentFactory
+      val viewComponent = viewComponentFactory.createViewComponent(this) as ViewComponentImpl
+      viewComponent.inject(this)
+    } catch (e: IllegalStateException) {
```

Comment:
we shouldn't need to catch this. Why is it needed?

Reply:
removing the try catch block